src/video_editor_mcp/server.py

```diff
@@ -276,6 +277,30 @@ def format_video_info_long(video):
     except Exception as e:
         return f"Error formatting video: {str(e)}"
     
+def format_local_searched_video_details(video):
+    # Get video duration in readable format
+    duration = str(timedelta(seconds=video.get('exif_info', {}).get('duration', 0)))
+
+    # Format date with timezone
+    video_date = video.get('date')
+    date_str = video_date.strftime("%B %d, %Y at %I:%M %p %Z") if video_date else "Date unknown"
+
+    # Get location details
+    place = video.get('place', {})
+    location = f"{place.get('name', 'Location unknown')}" if place else "Location unknown"
+
+    # Format detected faces
+    face_count = len(video.get('face_info', []))
+    faces = f"Detected {face_count} people" if face_count > 0 else "No people detected"
+
+    return (
+        f"Video: {video.get('filename', 'Untitled')}\n"
+        f"  📅 Recorded: {date_str}\n"
```

Comment:
Do the emojis help with claude's response? why add them? add a screenshot if they're beneficial

Reply:
ahh! Yes, i was expecting it to be be presented in such a way to user, but it doesn't seems so. Will close this